src/main/resources/regression/car17.yaml

```diff
@@ -0,0 +1,97 @@
+---
+name: car17
+index_command: target/appassembler/bin/IndexCollection
+index_utils_command: target/appassembler/bin/IndexUtils
+search_command: target/appassembler/bin/SearchCollection
+topic_root: src/main/resources/topics-and-qrels/
+qrels_root: src/main/resources/topics-and-qrels/
+index_root:
+ranking_root:
+generator: LuceneDocumentGenerator
+threads: 40
+index_options:
+  - -storePositions
+  - -storeDocvectors
+  - -optimize
+topic_reader: Car
+input: /tuna1/collections/car17/paragraphcorpus/
+index_path: "/tuna1/indexes/lucene-index.car17.pos+docvectors+rawdocs" # path to the existing index, used in regression test if `--index` option is absent
+collection: CarCollection
+index_stats:
+  documents: 29678360
+  documents (non-empty): 29674409
+  total terms: 1257896158
+topics:
+  - name: "All Topics"
+    path: topics.car17.test200.txt
+    qrel: qrels.car17.test200.hierarchical.txt
+evals:
+  - command: eval/trec_eval.9.0/trec_eval
+    params:
+      - -m map
+    separator: "\t"
+    parse_index: 2
+    metric: map
+    metric_precision: 4
+    can_combine: true
+  - command: eval/trec_eval.9.0/trec_eval
+    params:
+      - -m recip_rank
+    separator: "\t"
+    parse_index: 2
+    metric: recip_rank
+    metric_precision: 4
+    can_combine: true
+models:
+  - name: bm25
+    params:
+      - -bm25
+    results:
+      map:
+        - 0.1650
+      recip_rank:
+        - 0.2270
+  - name: bm25+rm3
+    params:
+      - -bm25
+      - -rm3
+    results:
+      map:
+        - 0.1354
+      recip_rank:
+        - 0.1860
+  - name: bm25+ax
+    params:
+      - -bm25
+      - -axiom
```

Comment:
In order to make the axiomatic reranking deterministic, you will need to add 
```
- -rerankCutoff 20
- -axiom.deterministic
```
Please see other yaml files for examples